packages/webapp-server/package.json

```diff
@@ -4,19 +4,23 @@
   "main": "src/index.ts",
   "license": "MIT",
   "scripts": {
-    "dev": "ts-node-dev src/index.ts",
+    "dev": "nodemon src/index.ts",
```

Comment:
Turns out nodemon will automatically work with ts-node if it's given a .ts file. ts-node-dev is not necessary.